shell/platform/windows/flutter_windows_engine.cc

```diff
@@ -45,7 +45,17 @@ FlutterRendererConfig GetRendererConfig() {
     }
     return host->view()->SwapBuffers();
   };
-  config.open_gl.fbo_callback = [](void* user_data) -> uint32_t { return 0; };
+  config.open_gl.fbo_reset_after_present = true;
+  config.open_gl.fbo_with_frame_info_callback =
+      [](void* user_data, const FlutterFrameInfo* info) -> uint32_t {
+    auto host = static_cast<FlutterWindowsEngine*>(user_data);
+    if (host->view()) {
+      return host->view()->GetFrameBufferId(info->size.width,
+                                            info->size.height);
+    } else {
+      return 0;
```

Comment:
can you add a comment on when this could happen? When can `host->view()` return null?